lib/core-generators/new/templates/assets/styles/components/ajax-button.component.less

```diff
@@ -0,0 +1,29 @@
+
+/**
+ * <ajax-button>
+ *
+ * App-wide styles for our ajax buttons.
+ */
+
+[parasails-component='ajax-button'] {
```

Comment:
@raqem tested it locally, and it looks like the loading styles aren't rendering on the button. In the component HTML, `.ajax-button` and `[parasails-component="ajax-button"]` are on the same level, so I think it's the nesting that's causing them not to apply.

Would you fix + re-test locally to make sure the loader is showing up? (Slowing down the network in the inspector helps you see the loading styles, I know it comes and goes pretty quickly otherwise)